pypesto/visualize/parameters.py

```diff
@@ -386,41 +380,30 @@ def handle_inputs(
     inner_xs = [
         res.get(INNER_PARAMETERS, None) for res in result.optimize_result.list
     ]
-    if any(inner_xs):
-        inner_xs_names = next(
-            list(inner_x.keys()) for inner_x in inner_xs if inner_x is not None
+
+    from ..hierarchical.calculator import HierarchicalAmiciCalculator
```

Comment:
Does this break the parameters plot for users without AMICI? Looks like imports are handled appropriately with `try-except` blocks in the different scripts this import depends on, but it would be important to ensure.

Reply:
Hmmm True. I also don't like how it's just there. It should be separated: in the sense that it should be imported only if there are non-None `inner_xs`.

Reply:
I changed it, and separated the inner_xs part into a separate function `handle_inner_inputs`.

Reply:
Looks nice! Thanks